### ISSUE TITLE
License issue for Debian packaging

### ISSUE THREAD

**Issue by rubund** [abk-openroad/OpenSTA#23]
Hi

Thanks for this great contribution to the open source world!

I noticed there is one problematic statement that prevents inclusion into Debian. Would it be possible to "clean up" this one?
See CMakeLists.txt:

```
# No part of this document may be copied, transmitted or
# disclosed in any form or fashion without the express
# written consent of Parallax Software, Inc.
```

Cheers

**Comment by jjcherry56** [comment]
This was fixed in ed3ad4f

**Comment by rubund** [comment]
Thanks!
